### PR TITLE
Adds documentation for monolog processors.

### PR DESCRIPTION
**Pull request by dbrumann**
Adds both a short description for existing processors as well as a note for the newly added TokenProcessor added in 3.4. I tried not to interfere with #10244 as this introduces documentation for more processors added in 4.3, so that PR hopefully can build on this one.

Replaces #8156

EUFOSSA

### CODE REVIEW

File: logging/processors.rst [hunk 1]
```diff
@@ -160,6 +160,15 @@ If you use several handlers, you can also register a processor at the
 handler level or at the channel level instead of registering it globally
 (see the following sections).
 
+Symfony's MonologBridge provides processors that can be registered inside your application.
+
+* :class:`Symfony\\Bridge\\Monolog\\Processor\\DebugProcessor` adds additional information useful for debugging like a timestamp or an error message to the record.
+* :class:`Symfony\\Bridge\\Monolog\\Processor\\TokenProcessor` adds information from the current user's token to the record namely username, roles and whether the user is authenticated.
+* :class:`Symfony\\Bridge\\Monolog\\Processor\\WebProcessor` overrides data from the request using the data inside Symfony's request object.
+
+.. versionadded:: 3.4
```
Comment (OskarStark) [inline logging/processors.rst]: Please add a blank line after the directive

File: logging/processors.rst [hunk 2]
```diff
@@ -160,6 +160,15 @@ If you use several handlers, you can also register a processor at the
...
+    The :class:``Symfony\\Bridge\\Monolog\\Processor\\TokenProcessor`` class was added in Symfony 3.4.
```
Comment (OskarStark) [inline logging/processors.rst]: Only one tick should be use for the class

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: logging/processors.rst (hunk 1 above)
Comment (dbrumann) [reply logging/processors.rst]: done

**Review by OskarStark — APPROVED** [review]

File: logging/processors.rst [hunk 3]
```diff
@@ -160,6 +160,16 @@ If you use several handlers, you can also register a processor at the
 handler level or at the channel level instead of registering it globally
 (see the following sections).
 
+Symfony's MonologBridge provides processors that can be registered inside your application.
+
+* :class:`Symfony\\Bridge\\Monolog\\Processor\\DebugProcessor` adds additional information useful for debugging like a timestamp or an error message to the record.
```
Comment (javiereguiluz) [inline logging/processors.rst]: Note for mergers ... please wrap these lines to make them 80-char long. Thanks.

**Review by javiereguiluz — APPROVED** [review]

**Pull request merged**